src/frontend/App/typer/totrinnskontroll.ts

```diff
@@ -23,4 +23,26 @@ export type TotrinnskontrollOpprettet = {
     opprettetTid: string;
 };
 
-export type TotrinnskontrollMedBegrunnelse = TotrinnskontrollOpprettet & { begrunnelse: string };
+export type TotrinnskontrollMedBegrunnelseOgÅrsaker = TotrinnskontrollOpprettet & {
+    begrunnelse: string;
+    årsakerUnderkjent?: ÅrsakUnderkjent[];
```

Comment:
Som nevnt for backend, så kan denne endres til not nullable (etter at backend er fikset og merget da)
Samtidig burde kanskje begrunnelse endres til nullable?